### PR TITLE
Replace the PDF file with a MD file

### PR DESCRIPTION
**Pull request by c64cosmin**
This PR is meant for removing the original PDF and replace it with a MD (markdown) file format that is friendlier with GitHub.

Reason: it facilitates editing in the future. It's easier to read even in console terminals only.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ Creating "mi_main.py"
 The omigen_py tool will generate the files ```schema.py``` and ```mi_main.py```.
 
 The file mi_main.py will be edited in order to implement the custom python provider.
-Please refer to [Script Provider Getting Started Guide](/doc/ScriptProviderGettingStarted.pdf) for more details.
+Please refer to [Script Provider Getting Started Guide](/doc/gettingStarted.pdf) for more details.
```
Comment (JumpingYang001) [inline README.md]: gettingStarted.md

**Review by JumpingYang001 — COMMENTED** [review]
one link typo.

**Comment by c64cosmin** [comment]
@JumpingYang001 Thank you, it was fixed!

File: doc/gettingStarted.md [hunk 2]
```diff
@@ -0,0 +1,337 @@
...
+to the library files that handle them. The simplest registry files contain a value for "LIBRARY" and a value for "CLASS".
+The registry file associates a provider (LIBRARY) with the classes (CLASS) that it has handlers for.
+The registry files are stored in a predetermined location and have a ".reg" extension.
+Depending on which options the OMI Server was built with and particular environment variables,
+the registry files could be in a number of places. If OMI Server was installed with a package,
+the registry files are placed in "/etc/opt/omi/conf/omiregister/".
+If OMI Server was configured with the "--dev" option and built from source,
+the registry files are placed at "[omi-source-root]/omi/Unix/output/etc/omiregister/".
+There are a number of directories within the omiregister directory. Each of these subdirectories represents a namespace.
+And each ".reg" file represents a single provider.
```
Comment (alinbalutoiu) [inline doc/gettingStarted.md]: Might be better to split this in multiple paragraphs.

File: doc/gettingStarted.md [hunk 3]
```diff
@@ -0,0 +1,337 @@
...
+It defines the structures that are used, and details which methods are implemented.
+Details about the schema.py file are outside of the scope of this document.
+The mi_main.py file contains some startup and initialization code, and provides the callbacks
+that the OMI Server uses to execute the provider.
+The author of the provider may edit the body of these methods to implement the provider.
+The OMI Server determines where to find these files with the LIBRARY parameter of the ".reg" file.
+The value of this parameter is the path to the mi_main.py and schema.py files for this provider.
+This path is relative to the Provider File location (see above). In the XYZ_Frog example,
+the XYZ_Frog.reg file tells the OMI Server to look for
+mi_main.py and schema.py at "[omi-source-root]/omi/Unix/output/lib/XYZ_Frog/".
```
Comment (alinbalutoiu) [inline doc/gettingStarted.md:340]: Same.

**Review by alinbalutoiu — APPROVED** [review]
Some minor nits, good to have but not really necessary. Looks good to me.

File: doc/gettingStarted.md (hunk 2 above)
Comment (c64cosmin) [reply doc/gettingStarted.md]: Done!

File: doc/gettingStarted.md (hunk 3 above)
Comment (c64cosmin) [reply doc/gettingStarted.md:340]: Done!

**Pull request merged**